challenge_one/README.md

```diff
@@ -0,0 +1,23 @@
+# Nethermind Bot Deploy and Update Monitor
+
+## Description
+
+This agent detects when Nethermind deploys or updates a Forta Bot
+
+## Supported Chains
+
+- Polygon
+
+## Alerts
+
+- FORTA-BOT-UPDATE
+  - Fired when a transaction comes from Nethermind's wallet address that contains the agentUpdate function call
+
+- FORTA-BOT-CREATE
+  - Fired when a transcation comes from Netherminds wallet address that contains the createAgent function call
```

Comment:
```suggestion
  - Fired when a transaction comes from Nethermind's wallet address that contains the `agentUpdate` function call

- FORTA-BOT-CREATE
  - Fired when a transaction comes from Nethermind's wallet address that contains the `createAgent` function call
```

You can use backticks when referring to things related to code,  only to look better in the Readme.(also fixed two small typos)

Reply:
Also, besides describing the alert, you should also include the Severity, Type, and Metadata fields.

You can have a look e.g. [here](https://github.com/NethermindEth/Forta-Agents/tree/main/Balancer-Bots/Large-Delegation#alerts) for an example.